docs/release/release-process.md

```diff
@@ -204,7 +204,7 @@ point to the freshly released stack versions.
   and [Stack
   guides](https://github.com/crossplaneio/crossplane/blob/master/docs/stacks-guide-gcp.md).
   * searching for `:master` will help a lot here
-  * `kubectl -k` statements will contain `?ref=master`, which should be updated to the release
+  * `kubectl -k` statements will contain `?ref=release-0.5`, which should be updated to the release
```

Comment:
this one should not be done, its the example of what to search for

Reply:
Oh, hah.